### ISSUE TITLE
Not supporting multiple PATCH method requests.

### ISSUE THREAD

**Issue by singh-virendra** [azagniotov/stubby4j#93]
I tried to put multiple patch requests in response.yaml, which is not working as expected(as it works for PUT and POST).

**response.yaml**
```
-
  request:
    method: PATCH
    url: /simulator/content/v1/url1
    file: response/service/failure-request.json
  response:
    status: 404

-
  request:
    method: PATCH
    url: /simulator/content/v1/url1
  response:
    status: 200
    headers:
      content-type: application/json
    file: response/service/success.json
```

**failure-request.json**
```
{
  "identifier": {
    "identifierTyp": "IDENTIFIER",
    "identifierVal": "1234567890"
  },
  "permissions": {
    "PERMISSION.PERMISSION1": {
      "value": "ABC"
    }
  },
  "extra": {
    "extraId": "54"
  }
}
```

**success.json**
```
{
  "description": "Queued",
  "transactionId": "123-456-789"
}
```

Expected: It should return the response with 200, if no body is present while making request to url `/simulator/content/v1/url1`, else if the body `failure-request.json` is present then it should return 404 response. 

Actual: it returns the status for which the entry is present first in the file `response.yaml`.

**Comment by azagniotov** [comment]
Fixed in v6.0.1

**Comment by singh-virendra** [comment]
It's not working as expected. Even already working flows stopped working now. Opening new issue with logs.
